fibonacci/fpc/code.pas

```diff
@@ -3,26 +3,15 @@
 uses
   SysUtils;
 
-function Fibonacci(n: Int32): Int32;
+function Fibonacci(n: UInt32): UInt32; inline;
```

Comment:
What does `inline` do here?

Reply:
Procedures that are declared inline are copied to the places where they are called. This has the effect that there is no actual procedure call, the code of the procedure is just copied to where the procedure is needed, this results in faster execution speed if the function or procedure is used a lot. It is obvious that inlining large functions does not make sense.